src/mongocxx/gridfs/bucket.cpp

```diff
@@ -31,8 +31,6 @@
 #include <mongocxx/options/index.hpp>
 #include <mongocxx/stdx.hpp>
 
-#include <mongocxx/config/private/prelude.hh>
```

Comment:
I'd vote for leaving this in. Though CONTRIBUTING.md does not specify prelude as being necessary for .cpp files, the only cpp files that do not include prelude are tests/examples. It might be useful to have prelude included in all source files.

Additionally, I think MONGOCXX_INLINE_NAMESPACE_BEGIN is most directly included through prelude.hh => prelude.hpp. Though it'd pick it up from other headers, having the more direct include seems preferable.

Reply:
Done